lib/y-richtext.coffee

```diff
@@ -2,6 +2,9 @@ misc = (require "./misc.coffee")
 BaseClass = misc.BaseClass
 Locker = misc.Locker
 Editors = (require "./editors.coffee")
+Picker = (require "../../quill/src/lib/picker")
+ColorPicker = (require "../../quill/src/lib/color-picker")
```

Comment:
don't you think it is possible to outsource this? I'd rather have y-richtext only for richtext (with binding to ot-types).